[2024-07-01T05:30:54.465+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-01T05:30:54.474+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: amazon_analytics_dag.tsk_is_file_in_s3_available manual__2024-07-01T05:30:36.342686+00:00 [queued]>
[2024-07-01T05:30:54.481+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: amazon_analytics_dag.tsk_is_file_in_s3_available manual__2024-07-01T05:30:36.342686+00:00 [queued]>
[2024-07-01T05:30:54.481+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2024-07-01T05:30:54.496+0000] {taskinstance.py:2330} INFO - Executing <Task(S3KeySensor): tsk_is_file_in_s3_available> on 2024-07-01 05:30:36.342686+00:00
[2024-07-01T05:30:54.503+0000] {standard_task_runner.py:63} INFO - Started process 57254 to run task
[2024-07-01T05:30:54.506+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'amazon_analytics_dag', 'tsk_is_file_in_s3_available', 'manual__2024-07-01T05:30:36.342686+00:00', '--job-id', '372', '--raw', '--subdir', 'DAGS_FOLDER/amazonanalytics.py', '--cfg-path', '/tmp/tmpimnk018p']
[2024-07-01T05:30:54.508+0000] {standard_task_runner.py:91} INFO - Job 372: Subtask tsk_is_file_in_s3_available
[2024-07-01T05:30:54.551+0000] {task_command.py:426} INFO - Running <TaskInstance: amazon_analytics_dag.tsk_is_file_in_s3_available manual__2024-07-01T05:30:36.342686+00:00 [running]> on host ip-172-31-19-147.us-west-2.compute.internal
[2024-07-01T05:30:54.637+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='deeptivarghese.education@gmail.com' AIRFLOW_CTX_DAG_OWNER='deeptivarghese' AIRFLOW_CTX_DAG_ID='amazon_analytics_dag' AIRFLOW_CTX_TASK_ID='tsk_is_file_in_s3_available' AIRFLOW_CTX_EXECUTION_DATE='2024-07-01T05:30:36.342686+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-01T05:30:36.342686+00:00'
[2024-07-01T05:30:54.638+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-01T05:30:54.651+0000] {baseoperator.py:400} WARNING - S3KeySensor.execute cannot be called outside TaskInstance!
[2024-07-01T05:30:54.651+0000] {s3.py:117} INFO - Poking for key : s3://cleaned-data-zone-csv-bucket5/response_data_01072024053040.csv
[2024-07-01T05:30:54.657+0000] {base.py:84} INFO - Using connection ID 'aws_s3_conn' for task execution.
[2024-07-01T05:30:54.657+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3_conn', conn_type='aws') credentials retrieved from login and password.
[2024-07-01T05:30:55.302+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-07-01T05:30:55.303+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-01T05:30:55.307+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=amazon_analytics_dag, task_id=tsk_is_file_in_s3_available, run_id=manual__2024-07-01T05:30:36.342686+00:00, execution_date=20240701T053036, start_date=20240701T053054, end_date=20240701T053055
[2024-07-01T05:30:55.362+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-01T05:30:55.383+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-01T05:30:55.389+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
